extensions/discovery-and-network/services.yaml

```diff
@@ -0,0 +1,294 @@
+      name: spec.loadBalancerClass
+      placeholder: spec.loadBalancerClass.placeholder
+      visibility: "$root.spec.type = 'LoadBalancer'"
+    - path: spec.externalName
+      name: spec.externalName
+      placeholder: spec.externalName.placeholder
+      visibility: "$root.spec.type = 'ExternalName'"
+    - path: spec.loadBalancerIP
+      name: spec.loadBalancerIP
+      placeholder: spec.loadBalancerIP.placeholder
+      visibility: "$root.spec.type = 'LoadBalancer'"
+    - path: spec.healthCheckNodePort
+      name: spec.healthCheckNodePort
+      placeholder: spec.healthCheckNodePort.placeholder
+      visibility: "$root.spec.type = 'LoadBalancer' and $root.spec.externalTrafficPolicy = 'Local'"
+    - path: spec.allocateLoadBalancerNodePorts
+      name: spec.allocateLoadBalancerNodePorts
+      visibility: "$root.spec.type = 'LoadBalancer'"
+    - path: spec.publishNotReadyAddresses
+      name: spec.publishNotReadyAddresses
+    - path: spec.sessionAffinityConfig.clientIP.timeoutSeconds
+      visibility: "$root.spec.sessionAffinity = 'ClientIP'"
+      name: Timeout seconds
+      placeholder: enter timeout seconds
+      inputInfo: Default value is 10800(for 3 hours)
+
+  list: |-
+    - name: Controlled By
+      source: metadata.ownerReferences
+      widget: ControlledBy
+      kindOnly: true
+    - name: Type
+      source: spec.type
+      sort:
+        compareFunction: $compareStrings($second, $first)
+    - name: Cluster IP
+      source: spec.clusterIP
+      sort: true
+    - name: Ports
+      source: >-
+        $count(spec.ports) ? $map(spec.ports, function($value) {$value.port =
+        $value.targetPort ? [$string($value.port), '/', $string($value.protocol)] ~>
+        $join('') : [$string($value.port), ':', $string($value.targetPort), '/',
+        $string($value.protocol)] ~> $join('') }) ~> $join(', ')  : '-'
+    - name: External IPs
+      source: >-
+        $count(status.loadBalancer.ingress) ? $map(status.loadBalancer.ingress,
+        function($value) {$value.ip ? $value.ip : $value.hostname}) ~> $join(', '):
+        ($count($item.spec.externalIPs) ? $item.spec.externalIPs ~> $join(', '):
+        '-')
+  dataSources: |-
+    apirules:
+      resource:
+        kind: APIRule
+        group: gateway.kyma-project.io
+        version: v1alpha1
+      filter: $item.spec.service.name = $root.metadata.name
+    subscriptions:
+      resource:
+        kind: Subscription
+        group: eventing.kyma-project.io
+        version: v1alpha1
+      filter: >-
+        $substringAfter($substringBefore($item.spec.sink, '.'), '://') =
+        $root.metadata.name
+  general: |-
+    resource:
+      kind: Service
+      version: v1
+    urlPath: services
+    scope: namespace
+    name: Services
+    category: Discovery and Network
+  presets: |-
+    - name: template
+      default: true
+      value:
+        spec:
+          selector: 
+            app: ''
+          type: 'ClusterIP'
+  translations: |-
```

Comment:
we decide to move all translation as plain string to form/details/etc so this section could be removed and moved directly